scripts/linters/pre_commit_linter.py

```diff
@@ -308,6 +313,21 @@ def _get_all_files_in_directory(dir_path, excluded_glob_patterns):
     return files_in_directory
 
 
+def _get_linter_type(linter_type):
+    """This function is used to return the type of linters to be used.
+
+    Args:
+        linter_type: str. Type of linter.
+
+    Returns:
+        list(str). The list of all linter types.
+    """
+    if linter_type:
+        return [linter_type]
+
+    return ['custom', 'third_party']
```

Comment:
This should be set as a global variable. Also, it should throw an exception if there isn't a match.

Reply:
Done. For exception part see comment below.